api.swagger.yaml

```diff
@@ -680,6 +1483,28 @@ paths:
               schema:
                 $ref: '#/components/schemas/location-uuid-list-schema'
       description: Search for locations.
+  /locations/list:
+    get:
+      tags:
+        - locations
+      parameters:
+        - description: Filter by label.
+          in: query
+          name: label
+          schema:
+            type: string
+        - $ref: '#/components/parameters/offsetParam'
+        - $ref: '#/components/parameters/limitParam'
+      responses:
+        '200':
+          description: List of locations matching the filter criteria.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/location-uuid-label-list-schema'
+        '404':
+          description: No matching locations were found.
```

Comment:
Another 404 that should be an empty array! No idea how I missed all of these on the previous review :(

Reply:
updated